extension/js/content_scripts/webmail/gmail-element-replacer.ts

```diff
@@ -294,12 +295,12 @@ export class GmailElementReplacer implements WebmailElementReplacer {
       }
     }
     // conversation top-right icon buttons
-    if (convoUpperIcons.length) {
+    if (convoUpperIconsContainer.length) {
       if (useEncryptionInThisConvo) {
-        if (!convoUpperIcons.is('.appended') || convoUpperIcons.find('.use_secure_reply').length) {
-          // either not appended, or appended icon is outdated (convo switched to encrypted)
-          this.addfcConvoIcon(convoUpperIcons, this.factory.btnWithoutFc(), '.show_original_conversation', () => {
-            convoUpperIcons.find('.gZ').trigger('click');
+        if (!convoUpperIconsContainer.is('.appended') || convoUpperIconsContainer.find(convoUpperIcons).length) {
+          this.addfcConvoIcon(convoUpperIconsContainer, this.factory.btnWithoutFc(), '.show_original_conversation', () => {
```

Comment:
`see original` button is hiding/showing when you perform some actions with thread, like starring messages, expanding/hiding messages etc.
But it should stay there constantly after it was added:

https://github.com/FlowCrypt/flowcrypt-browser/assets/1062093/8b0dd3aa-cd40-4d92-8281-b7883affdf01

Reply:
Thank you -- I found the problem of why such behavior occurs. When performing actions, the upper icon buttons container from Gmail gets re-injected, and then the 'see original' button gets injected as well. I'm having difficulty with troubleshooting it as those updates happen every sec like a "heartbeat" in Gmail. We could test such behavior by going to `Console` and manipulating the upper icon container with `$('div.hj').append('<span>test</span>')` and clicking the star icon will remove this as well.

I have fixed all this by finding another element (but it could not be so perfect).

Reply:
It's not flickering, but doesn't move along with other element, so it stays on the same place and overflows other elements, like this:

<img width="360" alt="Screenshot 2023-10-26 at 15 57 06" src="https://github.com/FlowCrypt/flowcrypt-browser/assets/1062093/17367a58-6e96-4b7b-a883-2c7e19225ea5">

It seems to be more complicated than I expected, then let's leave this flickering as it was before your latest change.

Also let's add check for not re-appending this button when it's already inserted - https://github.com/FlowCrypt/flowcrypt-browser/blob/d8b4dc6ccea9482405ebbbb5b38fb2c00aa74a09/extension/js/content_scripts/webmail/gmail-element-replacer.ts#L230
Here we remove `.show_original_conversation` element and then append it again. Need to add check in the beginning of `addfcConvoIcon`:
```js
if ($(containerSel).find(iconSel).length) { return }
```

Reply:
Yes, I also find it complicated. I'll revert the changes now and implement the suggested fix for the element not to remove/re-append. thank you!